test/rekt/features/broker/feature.go

```diff
@@ -118,32 +118,40 @@ func ManyTriggers() *feature.FeatureSet {
 
 	for _, testcase := range tests {
 
+		testcase := testcase // capture variable
```

Comment:
Do we know when we are moving to go 1.22? It looks like we won't need this anymore going forward: https://go.dev/blog/loopvar-preview

cc @pierDipi @creydr

Reply:
@pierDipi  @creydr gentle ping, should we go forward with this change?